doc/development/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -0,0 +1,418 @@
+# Implementing the SQL Dialect Adapter's Mandatory Classes 
+
+Each SQL Dialect adapter consists of two or more classes, depending on how standard-compliant the source behaves.
+The minimum implementation requires that you create the following mandatory classes:  
+  
+1. [`<Your_dialect_name>SqlDialect.java`](#creating-main-dialect-class)
+1. [`<Your_dialect_name>AdapterFactory.java`](#creating-adapter-factory)
+1. [`<Your_dialect_name>Adapter.java`](#creating-adapter)
+
+[Athena Virtual Schema Repository](https://github.com/exasol/athena-virtual-schema)
```

Comment:
this line here seems a bit out of context. I'm not sure if something is missing or the line should be removed.